apps/src/lib/tools/jsdebugger/JsDebugger.jsx

```diff
@@ -406,7 +425,7 @@ export const UnconnectedJsDebugger = Radium(React.createClass({
         >
           <span
             style={[
-              this.state.windowWidth <= 1275 && !this.state.watchersHidden && styles.hidden,
+              this.state.consoleWidth <= 345 && styles.hidden,
```

Comment:
Yes, only because I'd rather have the slider show than the text, although when testing this, it always seemed to work appropriately on load so the default was never used.